core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -750,7 +750,7 @@ private void fillTriggeredElements(Triggerable t,
                 // when we grabbed all triggerables that target children of the
                 // repeat entry (via initTriggerablesRootedBy). Hence skip them
                 if (!isRepeatEntryInit && t.isCascadingToChildren()) {
-                    updatedNodes = findCascadeReferences(target, updatedNodes);
+                    //updatedNodes = findCascadeReferences(target, updatedNodes);
```

Comment:
Sorry, didn't mean for you to comment this out... probably messes something up if you do